src/algebra/quaternion_basis.lean

```diff
@@ -0,0 +1,166 @@
+/-
+Copyright (c) 2021 Eric Wieser. All rights reserved.
+Released under MIT license as described in the file LICENSE.
```

Comment:
The style linter doesn't like this :shrug:

Reply:
It's because I pasted the wrong license!